packages/shared/src/modules/groups/libs/types/group-get-all-item-response-dto.type.ts

```diff
@@ -0,0 +1,9 @@
+type GroupGetAllItemResponseDto = {
+	createdAt: string;
+	id: number;
+	name: string;
+	permissions: { id: number; name?: string }[];
```

Comment:
Why name is optional 🤔 
```suggestion
	permissions: { id: number; name: string }[];
```